main.go

```diff
@@ -51,6 +55,26 @@ func connectDb() *sql.DB {
 	return db
 }
 
+func migrationUp() error {
+	driver, err := postgres.WithInstance(dbObject, &postgres.Config{})
+	if err != nil {
+		return fmt.Errorf("migrating up: %w", err)
+	}
+
+	m, err := migrate.NewWithDatabaseInstance(
+		"file://migrations", //HOW IS THE URL INSIDE THE CONTAINER?	WHAT IS RELATIVE/ABSOLUTE PATH?
```

Comment:
Let's talk about it in a call. It will be easier to show you. 

You copied the migrations to the container in the Dockerfile.